src/percy-agent-client/dom.ts

```diff
@@ -188,6 +187,11 @@ class DOM {
       } else if (!enableJavaScript && builtWithJs) {
         cloned!.remove()
       }
+
+      // Remove any lazy loading attributes, since they hang asset discovery
+      if (cloned!.getAttribute('loading')) {
```

Comment:
I just checked, and `removeAttribute` actually won't produce an error if the attribute doesn't exist. So I think it's safe to avoid the conditional and always remove the loading attribute